backend/svirt.pm

```diff
@@ -73,8 +73,9 @@ sub do_stop_vm {
             $self->run_cmd("$ps Remove-VM -Force -VMName $vmname");
         }
         else {
-            $self->run_cmd("virsh destroy $vmname");
-            $self->run_cmd("virsh undefine $vmname");
+            my $vmware_remote_vmm = get_var('VMWARE_REMOTE_VMM');
+            $self->run_cmd("virsh $vmware_remote_vmm destroy $vmname");
+            $self->run_cmd("virsh $vmware_remote_vmm undefine --snapshots-metadata $vmname");
```

Comment:
IMHO, this $vmware_remote_vmm usage will make the svirt backend looks like supporting only hyperv and vmware. Its real default support on general linux will be blurred. So does the latter similar changes for vmware. 

What about changes 'vmware_remote_vmm' to a general name like 'vmm_connection' or whatever you prefer (no matter it is a local or remote connection to hypervisor), and just sets it to VMWARE_REMOTE_VMM when VIRSH_VMM_FAMILY is vmware?

Reply:
Good idea @alice-suse, I changed the code according to your idea.